static/lang/pt-BR.po

```diff
@@ -1383,7 +1383,7 @@ msgstr "Imagem não encontrada: %s"
 #: source/common/modules/markdown-editor/renderers/readability.ts:39
 #, javascript-format
 msgid "Readability mode (%s)"
-msgstr ""
+msgstr "Modo de legibilidade"
```

Comment:
You are missing the `%s` placeholder here (for the actual algorithm used) — was this by accident or on purpose?

Reply:
Sorry. I forgot that. I'll correct it right away.

Reply:
The `%s` is very simple: If the translatable string contains these two characters – here, it's `Readability mode (%s)`, this means that the `%s` will be replaced with some other string while the app runs.

In this case, `%s` will be replaced with the readability algorithm that is being used. I.e., in the app, `Readability mode (%s)` will become `Readability mode (dale-chall)`.

You can put this placeholder wherever it makes sense. You could, in theory, put it in front of your translation if that makes grammatical sense. I think here you can just modify the translation as such: `Modo de legibilidade (%s)` (but I don't speak Portuguese, so think whether `Modo de legibilidade (dale-chall)` makes grammatical sense first!)

Reply:
I guess I got it right, now. Would please let me know? Tks.